getting_started/setup_linux/prepare_build_environment.rst

```diff
@@ -0,0 +1,104 @@
+.. _prepare-build-environment:
+
+Set up a development environment to build the kernel
+####################################################
+
+.. contents::
+   :local:
+   :depth: 3
+
+Introduction
+************
+
+
+These instructions will help you set up a development environment for the SOF branch of the Linux kernel. If you have dedicated test hardware you can use ktest to install it over ssh, otherwise you can install it locally on your device in addition to your default kernel.
```

Comment:
SOF branch or SOF fork?

Reply:
While it technically is a fork, I want to avoid scaring people by making them think they're installing some weird unofficial variant of Linux. While that's obviously not the case, some people are very protective of their systems yet know nothing about them, so I figured this is best.